runtime/convertValues_test.go

```diff
@@ -2170,19 +2170,37 @@ func TestMalformedArgumentPassing(t *testing.T) {
 			expectedErrType: &MalformedValueError{},
 		},
 		{
-			label:         "Array with malformed member",
+			label:         "Variable-size array with malformed element",
 			typeSignature: "[Foo]",
 			exportedValue: cadence.NewArray([]cadence.Value{
 				malformedStruct1,
 			}),
 			expectedErrType: &MalformedValueError{},
 		},
 		{
-			label:         "Array with wrong size",
-			typeSignature: "[String; 2]",
+			label:         "Constant-size array with malformed element",
+			typeSignature: "[Foo; 1]",
 			exportedValue: cadence.NewArray([]cadence.Value{
 				malformedStruct1,
```

Comment:
oh so this was failing before reaching the size check (which wasn't there), because the inner element is malformed?

Reply:
Correct, the test was failing, but for the "wrong reason"